src/backend/columnar/columnar_tableam.c

```diff
@@ -1910,6 +1910,11 @@ ColumnarSubXactCallback(SubXactEvent event, SubTransactionId mySubid,
 void
 columnar_tableam_init()
 {
+	ColumnarTableSetOptions_hook_type **ColumnarTableSetOptions_hook_ptr =
+		(ColumnarTableSetOptions_hook_type **) find_rendezvous_variable(
+			"ColumnarTableSetOptions_hook");
```

Comment:
small nit; do columnar and citus share a header file somewhere? ifso, would it make sense to extract `"ColumnarTableSetOptions_hook"` to that header file? Currently this constant seems to be declared twice while its semantics are the same.

Most of the time we use defined constants for those.

Reply:
Hmm... since it's matching a symbol name it didn't seem to quite make sense -- it's kind of incidentally a string but it seems like it should match the name of the symbol. I don't have a strong opinion here.